OutOfSchool/OutOfSchool.AuthCommon/Services/ProviderAdminService.cs

```diff
@@ -103,15 +101,10 @@ public class ProviderAdminService : IProviderAdminService
                         userId,
                         result.ErrorMessages());
 
-                    response.IsSuccess = false;
-                    response.HttpStatusCode = HttpStatusCode.BadRequest;
-
                     // TODO: Don't leak all the errors eventually
-                    response.Message = string.Join(
-                        Environment.NewLine,
-                        result.Errors.Select(e => e.Description));
-
-                    return response;
+                    return CreateResponseDto(
+                        HttpStatusCode.BadRequest,
+                        string.Join(Environment.NewLine, result.Errors.Select(e => e.Description)));
```

Comment:
(Another legacy issue) There is `IdentityResult` extension method `.ErrorMessages()` that could be used instead of
`string.Join(Environment.NewLine, result.Errors.Select(e => e.Description))`

Suggestions:
1. Please use it (there are 5 places in this file), 
2. For this specific place in file please cache the `result.ErrorMessages()` as it is used in both log message parameters and `CreateResponseDto()`

Reply:
Fixed. Thank you!

Reply:
Please check comment below